inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImpl.java

```diff
@@ -386,18 +386,25 @@ public void deleteConcept(KnowledgeBase kb, KBConcept aConcept)
     @Override
     public KBHandle createProperty(KnowledgeBase kb, KBProperty aProperty)
     {
-        if (StringUtils.isNotEmpty(aProperty.getIdentifier())) {
+        if (StringUtils.isNotEmpty(aProperty.getIdentifier())
+                && (!aProperty.getIdentifier().equals(kb.getSubclassIri().stringValue()))) {
             throw new IllegalArgumentException("Identifier must be empty on create");
         }
-
         return update(kb, (conn) -> {
-            String identifier = generateIdentifier(conn, kb);
+            String identifier;
+            if (aProperty.getIdentifier() != null && aProperty.getIdentifier().toString()
+                    .equals(kb.getSubclassIri().stringValue())) {
```

Comment:
Same here. Why is there an exception for the subclass IRI? There is no comment explaining why this condition is used.